pkg/test/resourcefixture/testdata/basic/alloydb/v1beta1/alloydbcluster/fullalloydbcluster/create.yaml

```diff
@@ -51,3 +51,12 @@ spec:
   encryptionConfig:
       kmsKeyNameRef: 
         name: kmscryptokey-${uniqueId}
+
+  maintenanceUpdatePolicy:
+    maintenanceWindows:
+      - day: WEDNESDAY
+        startTime:
+          hours: 12
+          minutes: 0
+          seconds: 0
+          nanos: 0
```

Comment:
Also one newline at the bottom is always recommended.

Reply:
Done